### DISCUSSION TITLE
Student Progress Tracker

### DISCUSSION THREAD

**Discussion by Darshan-Limbani** [scaleracademy/open-source-projects#109] [Ideas]
# It is a system for a college student. 
It include all the student related activity of the colleges. it contains the following things:
- Student Attendance
- Fees Payment
- Generate Marksheet
- Schedule for different Activities
- College Notice Board
- Monthly Report based on the Attendance and the other co-curricular Activities
- Support Section for the Student. etc...

**Comment by badielubiin** [comment]
A support section would be great.

**Reply by Darshan-Limbani** [reply]
yah. I added that thank you for your Suggestion.

**Comment by kuraakhilesh8230** [comment]
A Behavior data would be great

**Comment by yakin18** [comment]
great for student profile.

**Comment by jaypansuriya104** [comment]
It's amazing idea As it's very helpful for students to enhance their Capabilities , Creativities , Productivity !

**Comment by YuvrajBaser** [comment]
There can be a section where students can compete with each other by organizing coding contests, quiz, and various tests related to subject.

**Reply by Darshan-Limbani** [reply]
Yes, it is also a good idea which can be implemented As Overall it is all about the student related activities at a single platform.

**Comment by imhardikdesai** [comment]
Very Beneficial for students. It like Student community, Great idea 👍🏻

**Comment by URVESH1121** [comment]
Great idea, Students also manage their data easily.👍

**Comment by anisnorn** [comment]
Your provided system help me alot, when I go [apartments for rent in saida](https://www.beiteck.com/rental) , thank you

**Comment by Jamesbond2314** [comment]
The [Sonic Birthday Treat](https://sonicsmenu.us/sonic-birthday-reward/) offers a fun and festive way to celebrate special occasions with a delicious variety of customizable desserts and drinks. From refreshing slushes to delectable cakes, Sonic ensures that every birthday celebration is filled with tasty treats that make the day even more memorable.
